control-plane/roles/auditing-meili/defaults/main/main.yaml

```diff
@@ -0,0 +1,5 @@
+---
+metal_auditing_meili_ingress: {}
+metal_auditing_meili_environment: development
+metal_auditing_meili_persistence: {}
+metal_auditing_meili_chart_version: "0.1.44"
```

Comment:
We can use this value from the release vector instead. So this can be removed and in `tasks/main.yaml` we should add a task for resolving the release vector through the `setup_yaml` module.